### ISSUE TITLE
Archetypes fails due to Java 8 

### ISSUE THREAD

**Issue by AIMMOTH** [GoogleCloudPlatform/app-maven-plugin#197]
Creating either 2, 3 or 5 fails since they use Java 8 
> mvn archetype:generate -Dappengine-version=1.9.52 -Dapplication-id=your-app-id -Dfilter=com.google.appengine.archetypes: 
... 
1: remote -> com.google.appengine.archetypes:appengine-flexible-archetype (A basic Java application with Google App Engine flexible.)
2: remote -> com.google.appengine.archetypes:appengine-skeleton-archetype (A skeleton application with Google App Engine)
3: remote -> com.google.appengine.archetypes:appengine-standard-archetype (A basic Java application with Google App Engine Standard)
4: remote -> com.google.appengine.archetypes:endpoints-skeleton-archetype (A skeleton project using Cloud Endpoints Frameworks with Google App Engine Standard)
5: remote -> com.google.appengine.archetypes:guestbook-archetype (A guestbook application with Google App Engine)
6: remote -> com.google.appengine.archetypes:hello-endpoints-archetype (A simple starter application using Cloud Endpoints Frameworks with Google App Engine Standard)
7: remote -> com.google.appengine.archetypes:skeleton-archetype (-) 

> mvn appengine:deploy
...
[INFO] GCLOUD: ERROR: (gcloud.app.deploy) INVALID_ARGUMENT: Invalid runtime 'java8' specified. Accepted runtimes are: [go, java, java7, php, php55, python, python27]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 19.880 s
[INFO] Finished at: 2017-06-22T08:38:21+02:00
[INFO] Final Memory: 19M/306M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.google.cloud.tools:appengine-maven-plugin:1.3.1:deploy (default-cli) on project gae-soap-service: Execution default-cli of goal com.google.cloud.tools:appengine-maven-plugin:1.3.1:deploy failed: Non zero exit: 1 -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException

**Comment by patflynn** [comment]
@lesv Have we shipped appengine archetypes using Java 8?

**Comment by lesv** [comment]
Yes - You should modify the default parameters and say Java 8 false.

Unfortunately, archetypes take a week to release, so they are a bit ahead of things.

**Comment by lesv** [comment]
`mvn archetype:generate -Djava8=false -Dappengine-version=1.9.52 -Dapplication-id=your-app-id -Dfilter=com.google.appengine.archetypes:`
